CHANGES.md

```diff
@@ -9,6 +9,7 @@
 - Add primer support and test for code piped into black via STDIN (#2315)
 - Fix internal error when `FORCE_OPTIONAL_PARENTHESES` feature is enabled (#2332)
 - Accept empty stdin (#2346)
+- Moved from `appdirs` dependency to `platformdirs` (#2375)
```

Comment:
Can you merge with main and put this in a new section? We just made a release.

Reply:
Done 👍